### DISCUSSION TITLE
Unable to fly high

### DISCUSSION THREAD

**Discussion by liu-94** [USC-ACTLab/crazyswarm#622] [General]
the crazyflie is unable to fly above 2m, it became unstable at near 2m mark and crashes, may i know why?

i am using LPS system + flowdeck with crazyswarm and the nodes are set up in an 5 x 5 x 2.5m space

Thank you

**Comment by knmcguire** [comment]
Hi! 

Are you flying with the flowdeck 1 or 2? Flowdeck2 should be able to handle it but at 2 meters there better be good enough texture and lighting for the flowdeck to still see optical flow. Also make sure that you are not flying above full black areas and low lit grounds. 

You could also increase the standard deviation of the flowdeck as well, which can be done with the [motion.flowStdFixed](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/params/#motion) parameter. If you are feeling adventurous you can also go ahead and try the motion.adaptive parameter, but do it with caution as it is experimental
